src/slack

```diff
@@ -35,9 +34,10 @@ esac
 
 # ARGUMENT AND OPTION PARSING #####################################################################
 while (( "$#" )); do
+  echo "${1}"
   case "${1}" in
     --actions=*) actions=${1/--actions=/''} ; shift ;;
-    --actions*|-a*) actions=${2} ; shift ; shift ;;
+    --actions*|-ac*) actions=${2} ; shift ; shift ;;
```

Comment:
`-a*` here was masking `-ai*` from `--author-icon`, might instead want to remove one or the other...